### ISSUE TITLE
Make urls clickable in server push response

### ISSUE THREAD

**Issue by staeke** [sublimehq/sublime_merge#42]
Bitbucket, as well as other hosting services I’m sure, sends a response text on git push that includes the url for making a pull request. If I do git push in iTerm2 it parses the link and I can cmd+click it to make a PR. It’d be awesome if Sublime Merge did the same on messages received so when I pull up the server response I can just click the link. Bonus points for some even nicer ui

**Comment by dhempy** [comment]
Yes, please!

**Comment by franksands** [comment]
This would be very useful. A lot of git clients do this currently

**Comment by arianf** [comment]
I currently use `git open` (https://github.com/paulirish/git-open) in when in terminal to do this in the command line, it would be great if sublime merge supported sometime like this. Not all Git clients respond with a fancy url, but the url structure is pretty simple, and can be piece together using the remote origin url.

**Comment by dougis** [comment]
another vote for this from me

**Comment by mkunzmann** [comment]
another vote. This would also be extremely helpful when working with gerrit.

**Comment by jskinner** [comment]
URLs are highlighted, and can be opened from the context menu in build 2020

**Comment by howeaj** [comment]
Note: to open them, you have to right click them and select "open"

**Comment by dhempy** [comment]
@howeaj @jskinner  - Where exactly do you do this? I've right-clicked on the branch, the commit, explored the menus, and elsewhere that seemed likely ... and I have not found the "Open" option, nor the output from `git push` output. 

Can you ELI5, please?

I'm on build 2032.

![image](https://user-images.githubusercontent.com/966566/92513723-c21dfb80-f1de-11ea-8380-bc1366d9fb79.png)

**Comment by franksands** [comment]
> @howeaj @jskinner - Where exactly do you do this? I've right-clicked on the branch, the commit, explored the menus, and elsewhere that seemed likely ... and I have not found the "Open" option, nor the output from `git push` output.
> 
> I'm on build 2032.
> 
Hi, you have to right click on the link from the push response and choose "Open ..." from the context menu that appears
<img width="920" alt="Screen Shot 2020-09-08 at 15 36 14" src="https://user-images.githubusercontent.com/16136780/92515175-8ccadb00-f1e9-11ea-82ba-7d9f7ddc1162.png">

**Comment by dhempy** [comment]
Hmm...how can I see the push response? After I push, I don't see any push response message, as you show in your screenshot. I've gone through all the menus and haven't found how to see that. Am I missing something obvious? What build are you on, @franksands ?

**Comment by dhempy** [comment]
I'm on a Mac, FWIW. It appears I'm running the most recent version:
![image](https://user-images.githubusercontent.com/966566/93477814-78619f00-f8c9-11ea-898a-1d681f27b3df.png)

**Comment by srbs** [comment]
@dhempy, click on the icon to the left of the branch name. Look like this with a border and green check: `> ✓`

You can also always show command output by setting the preference `Advanced` > `Show Command Status` > `Always`

**Comment by dhempy** [comment]
Thank you, @srbs !  That was the piece I was overlooking: 
![image](https://user-images.githubusercontent.com/966566/93632037-a4a91880-f9ba-11ea-871f-3c895e0fbea0.png)

I also changed the preferences setting you suggested. Thanks! 👍

**Comment by dhempy** [comment]
FWIW, when setting preference Advanced > Show Command Status > Always , the response message shown does not have clickable links. They are only clickable when you view the message from the status icon, pictured above.

**Comment by howeaj** [comment]
@jskinner Given the large amount of confusion here from the currently implemented solution, I'd consider reopening this to implement the more intuitive solution of just a left click to open the link, in both the status-icon and always-show versions of the message.

**Comment by srbs** [comment]
@howeaj, I would recommend creating a new issue for that.
